### ISSUE TITLE
Why is there no validation loss?

### ISSUE THREAD

**Issue by ackbar03** [sthalles/SimCLR#31]
Hi, thanks very much for providing the code and framework!

Can I check why the code does not monitor loss evolution for a separate validation set? I see from the closed issues that there used to be one, but it seems to have been removed in latest. I'm sure validation loss should still be monitored to ensure model learns proper features right?

Thanks!
Michael

**Comment by here101** [comment]
I think the evaluation code may be the ipynb in the feature_eval folder,and the ipynb need to be run in the google colab.
